Documentation/guides/BuildProcess.md

```diff
@@ -360,6 +377,14 @@ when packaing Release applications.
     <AndroidLinkSkip>Assembly1;Assembly2</AndroidLinkSkip>
     ```
 
+-   **AndroidLinkTool** &ndash; An enum-style property with valid
```

Comment:
It feels "weird" that one property has a `Tool` suffix, while the other has a `Generator` suffix.  Should these be more consistent?  Should we call them e.g. `$(AndroidDexTool)`/`$(AndroidLinkTool)` or `$(AndroidDexGenerator)`/`$(AndroidLinkGenerator)`?  (The latter pair looks bad, but the former looks interesting...)

Reply:
It's now renamed to `$(AndroidDexTool)`